File: Day 13/Aayushi-Mittal.cpp
// To find whether a number is armstrong or not.

#include <iostream>
#include<math.h>
using namespace std;
int main()
{
  long long int num=0, ld=0, count, sum=0;
    cout<<"enter a number to check armstrong number : "<<endl;
    cin>> num;
    while (num>0)
    {
        num=num/10;
        count++;
    }
    while (num>0)
    {
        num=num/10;
        ld=num%10;
        sum=sum+pow(ld,count);
    }
    if(sum==num)
    cout<<"it is an armstrong number."<<endl;
    else
    cout<<"it is not an armstrong number.";
    return 0;
}

File: Day 13/Aayushi-Mittal.py
# To find whether a number is armstrong or not.
# An Armstrong number is a n-digit number that is equal to the sum of the nth power of its digits.

ld=0 
count=0 
sum=0
print("enter a number to check armstrong number : ")
num = int(input())
while num>0:
    num=num/10
    count++
while num>0:
    num=num/10
    ld=num%10
    sum=sum+ld**count
if sum==num:
    print("it is an armstrong number.")
else:
    print("it is not an armstrong number.")
